semcore/dropdown/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [6.36.2] - 2024-08-20
+
+### Fixed
+
+- Focusing in popover.
```

Comment:
```suggestion
- Focusing in popper.
```
?

Reply:
No, popper might stand either for "poped" content or the context container, that includes both trigger and popper, while with popover we always refer to the "popped" part